sqldelight-gradle-plugin/src/test/kotlin-mpp/build.gradle

```diff
@@ -24,7 +24,17 @@ kotlin {
   sourceSets {
     commonMain {
       dependencies {
-        implementation 'org.jetbrains.kotlin:kotlin-stdlib'
+        implementation deps.kotlin.stdlib.common
+      }
+    }
+    jvmMain {
+      dependencies {
+        implementation deps.kotlin.stdlib.jdk
```

Comment:
idk the sqldelight runtime dependency works but this doesnt. Also the only thing it was failing on (for js and jvm) was `trimMargin` not being a thing. ???